vj4/ui/pages/record_detail.page.js

```diff
@@ -0,0 +1,21 @@
+import { NamedPage } from 'vj/misc/PageLoader';
+
+const page = new NamedPage('record_detail', async () => {
+  const SockJs = await System.import('sockjs-client');
+  const DiffDOM = await System.import('diff-dom');
+
+  const sock = new SockJs(window.location.pathname + '/conn');
```

Comment:
Better to define connection endpoint in a JavaScript variable in HTML `<script>` tag (e.g. like `Context`)